std/concurrency.d

```diff
@@ -765,6 +747,42 @@ do
         return ret;
 }
 
+///
+@system unittest
```

Comment:
Why the `@system`?

Reply:
I tried `@safe` but `spawn` turned out to be system, though I haven't investigated further.

Reply:
Yes but why the _explicit_ @system?

Reply:
So that they can be found easily via grep and there is an active awareness for both the submitter and reviewers. That's why there is a rule that checks that a unittest is annotated with either `@safe` or `@system`

Reply:
I did not know about that rule. I don't think I've ever ran into it before.